core/src/database/configuration/organization_configuration_key/organization_configuration_key.enum.ts

```diff
@@ -0,0 +1,13 @@
+import {registerEnumType} from "@nestjs/graphql";
+
+export enum OrganizationConfigurationKeyCode {
```

Comment:
One of these things is not like the others. This enum name doesn't reflect its contents (and it's super long at that).

Reply:
This leads to stuff like OrganizationConfigurationKeyProviderFactory.java